libs/expo/betterangels/src/lib/screens/Clients/Clients.graphql

```diff
@@ -29,37 +29,44 @@ mutation CreateNote($data: CreateNoteInput!) {
   }
 }
 
-query ClientProfiles(
+query ClientProfilesPaginated(
   $filters: ClientProfileFilter
   $pagination: OffsetPaginationInput
   $order: ClientProfileOrder
 ) {
-  clientProfiles(filters: $filters, pagination: $pagination, order: $order) {
-    ... on ClientProfileType {
-      id
-      age
-      dateOfBirth
-      heightInInches
-      mailingAddress
-      nickname
-      residenceAddress
-      hmisProfiles {
-        id
-        agency
-        hmisId
+  clientProfilesPaginated(filters: $filters, pagination: $pagination, order: $order) {
+    ... on ClientProfileTypeOffsetPaginated {
+      pageInfo {
+          offset,
+          limit
       }
-      profilePhoto {
-        name
-        url
-      }
-      user {
+      totalCount
+      results {
         id
-        email
-        firstName
-        lastName
-        username
+        age
+        dateOfBirth
+        heightInInches
+        mailingAddress
+        nickname
+        residenceAddress
+        hmisProfiles {
+            id
+            agency
+            hmisId
+        }
+        profilePhoto {
+            name
+            url
+        }
+        user {
+            id
+            email
+            firstName
+            lastName
+            username
+        }
+        displayCaseManager
       }
-      displayCaseManager # TODO: This shouldn't be here. Temporary fix while we figure out type generation.
```

Comment:
TODO comments should be left in unless you're certain that the thing they're describing has been resolved